dailymed-ui/src/components/RxTermsSearch/index.jsx

```diff
@@ -97,46 +63,87 @@ export default class RxTermsSearch extends React.Component {
     render() {
         const { settings, options, selected } = this.state;      
         return (
-            <div>
-                <Form.Group>
-                    <Form.Label>Drug name</Form.Label>
-                    <Typeahead
-                        id="drug-name-typeahead"
-                        minLength={settings.drugNameMinLength}
-                        highlightOnlyResult={settings.highlightOnlyResult}
-                        paginate={settings.paginate}
-                        maxResults={settings.maxResults}
-                        labelKey={'displayName'}
-                        onInputChange={(text) => this.handleDrugNameChange(text)}
-                        onChange={(selection) => this.setDrugName(selection)}
-                        options={options.drugName}
-                        placeholder="Enter a drug name"
-                    />
-                    <Form.Text>Minimum {settings.drugNameMinLength} characters.</Form.Text>
-                </Form.Group>
-
-                <Form.Group>
-                    <Form.Label>Drug strength and form</Form.Label>
-                    <Typeahead
-                        id="drug-strength-typeahead"
-                        minLength={0}
-                        highlightOnlyResult={settings.highlightOnlyResult}
-                        paginate={settings.paginate}
-                        maxResults={settings.maxResults}
-                        labelKey={'strengthAndForm'}
-                        onInputChange={(text) => this.handleDrugStrengthAndFormChange(text)}
-                        onChange={(selection) => this.setDrugStrengthAndForm(selection)}
-                        options={options.drugStrengthAndForm}
-                        placeholder="Choose a strength and form"
-                    />
-                </Form.Group>
-
-                <div className="float-right">
-                    <Button variant="light">Reset</Button>{' '}
-                    <Button variant="primary" disabled={!selected.rxcui}>Next</Button>{' '}
-                </div>
-
-            </div>      
+          <div>
+            <Form.Group>
+              <Form.Label>Drug name</Form.Label>
+              <AsyncTypeahead
+                id="drug-name-typeahead"
+                minLength={settings.drugNameMinLength}
```

Comment:
@davidhicks980 - I'm getting an error (below).  You need to add the following labelKey property to your Drug name AsyncTypeahead:  `labelKey={"displayName"}`

`Fields passed to `filterBy` should have string values. Value will be converted to a string; results may be unexpected. `